.gitignore

```diff
@@ -5,3 +5,12 @@
 /ecs-init-*
 *.swp
 /ecs.conf
+/.rpm-done
+/.srpm-done
+/BUILD
+/RPMS
+/SOURCES
+/SRPMS
+/ecs-init.spec
+/sources.tgz
+ecs-init/version/version.go
```

Comment:
I must have missed the discussion on moving away from committing the generated `version.go`, is there a good reason to not commit the file when it changes even if it is generated?